packages/plugin-conversation/src/conversation.js

```diff
@@ -205,10 +287,20 @@ const Conversation = SparkPlugin.extend({
     });
   },
 
+  /**
+   * List activities for the specified conversation
+   * @param {Object} options
+   * @returns {Promise<Array<Activity>>}
+   */
   listActivities(options) {
```

Comment:
Should we have `listActivities` and `listMentions` throw errors if a conversation isn't provided? Maybe something to add in another PR